<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClockWindow</class>
 <widget class="QMainWindow" name="ClockWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClockWindow</string>
  </property>
  <widget class="SideBarWidget" name="centralwidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actiontoggle_bar"/>
   <addaction name="actionpop_creator_window"/>
  </widget>
  <action name="actiontoggle_bar">
   <property name="text">
    <string>toggle_bar</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionpop_creator_window">
   <property name="text">
    <string>pop_creator_window</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SideBarWidget</class>
   <extends>QWidget</extends>
   <header>sidebarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
